lib/util.js

```diff
@@ -1,5 +1,6 @@
-import fs from 'node:fs';
+import fs, { close, openSync, statSync, utimesSync } from 'node:fs';
 import { EOL } from 'node:os';
+import { execSync } from 'node:child_process';
```

Comment:
This import doesn't seem to be used (found this by running `npm run lint`)

Reply:
Fixed